kraken-build/src/kraken/std/python/buildsystem/uv.py

```diff
@@ -259,17 +253,14 @@ def build_v2(self, settings: PythonSettings, output_directory: Path) -> list[Pat
                 "--installer",
                 "uv",
             ]
-            logger.info(
-                "Running %s in '%s' with env %s",
-                command,
-                self.project_directory,
-                sanitize_http_basic_auth(str(env)),
-            )
-            sp.check_call(command, cwd=self.project_directory, env={**os.environ, **env})
+            logger.debug("Run '%s' in %s.", " ".join(command), self.project_directory)
```

Comment:
logging level change spotted.  (here and  line 223). In the other files they were `info`. Why the change to `debug`?